packages/salesforcedx-utils-vscode/src/test/orgUtils.ts

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { CliCommandExecutor, CommandOutput, SfdxCommandBuilder } from '../cli';
+import childProcess = require('child_process');
+import * as path from 'path';
+import * as util from 'util';
+
+// Used only for CI purposes. Must call delete if you call create
+export async function createSFDXProject(projectName: string): Promise<void> {
+  const execution = new CliCommandExecutor(
+    new SfdxCommandBuilder()
+      .withArg('force:project:create')
+      .withFlag('--projectname', projectName)
+      .withJson()
+      .build(),
+    { cwd: process.cwd() }
+  ).execute();
+  const cmdOutput = new CommandOutput();
+  await cmdOutput.getCmdResult(execution);
+  return Promise.resolve();
+}
+
+export async function createScratchOrg(projectName: string): Promise<string> {
+  const scratchDefFilePath = path.join(
+    process.cwd(),
+    projectName,
+    'config',
+    'project-scratch-def.json'
+  );
+  const execution = new CliCommandExecutor(
+    new SfdxCommandBuilder()
+      .withArg('force:org:create')
+      .withFlag('--definitionfile', `${scratchDefFilePath}`)
+      .withArg('--setdefaultusername')
+      .withJson()
+      .build(),
+    { cwd: path.join(process.cwd(), projectName) }
+  ).execute();
+  const cmdOutput = new CommandOutput();
+  const result = await cmdOutput.getCmdResult(execution);
+  const username = JSON.parse(result).result.username;
+  return Promise.resolve(username);
+}
+
+export async function deleteScratchOrg(
```

Comment:
@Sweetman Added a deleteScratchOrg method so that cleanup after a test can happen.